libavcodec/qsv.c

```diff
@@ -79,7 +79,8 @@ static const struct {
     MAP(MPEG2VIDEO,  MPEG2_HIGH,      MPEG2_HIGH   ),
 
     MAP(H264,        H264_BASELINE,   AVC_BASELINE ),
-    MAP(H264,        H264_CONSTRAINED_BASELINE, AVC_BASELINE),
+    MAP(H264,        H264_CONSTRAINED_BASELINE,
+                                      AVC_BASELINE ),
```

Comment:
please don't miss such thing in one patch.  One separated patch will be better.

Reply:
Sure, thanks.